src/pages/CreateReferral/index.tsx

```diff
@@ -419,7 +418,6 @@ export default function CreateReferral() {
           </Flex>
         </Flex>
       </BodyWrapper>
-      <NetworkModal />
```

Comment:
any reason behind this? is it redundant?

Reply:
yes, redudant, I see 2 modals in our HTML. 
I read the code and see this is a global modal.